bot.py

```diff
@@ -338,6 +338,147 @@ async def display_current_list_of_summoners(ctx):
         await ctx.send(embed=create_embed(embed_data))
 
 
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+@bot.command(name="remove", help="Remove player(s) from the list")
+async def remove_summoner(ctx, *, message):
+    """Remove summoner(s) from list
+    and send  the list to the bot"""
+
+    try:
+        # typing indicator
+        async with ctx.typing():
+            await asyncio.sleep(1)
+
+        # converting the message into list of summoners
+        summoner_to_remove_input = message.split(",")
+
+        # Exception case: attempt to remove more than 10 players
+        if len(summoner_to_remove_input) > MAX_NUM_PLAYERS_TEAM:
+            raise Exception (
+                "Limit Exceeded",
+                "You have exceeded a limit of 10 summoners! \
```

Comment:
`You have exceeded a limit of 10 summoners!`
Doesn't make sense. 
should be more like 'You tried to remove more than 10 summoners!'